src/components/authorization/policies/by-role/field-partner.policy.ts

```diff
@@ -1,6 +1,29 @@
-import { member, Policy, Role, variant } from '../util';
+import { inherit, member, Policy, Role, variant } from '../util';
 
 @Policy(Role.FieldPartner, (r) => [
+  r.Ceremony.read,
+  inherit(
+    r.Engagement.read,
+    r.InternshipEngagement.when(member).edit.specifically(
+      (p) => p.ceremony.none
+    ),
+    r.LanguageEngagement.when(member).read
+  ),
+  r.EthnologueLanguage.when(member).read,
+  r.FieldRegion.read,
+  r.FieldZone.read,
+  r.FinancialReport.read,
+  r.Language.when(member).read,
+  r.Organization.when(member).read.specifically((p) => p.address.none),
+  r.Partner.when(member).read.specifically((p) => p.pointOfContact.none),
+  r.Partnership.when(member).read,
+  r.PeriodicReport.read,
```

Comment:
Unless we are trying to also grant read access to `NarrativeReports` this isn't needed here as the two other concretes are specified. Or we could leave this and remove FinancialReport entry.

Reply:
Nice catch...removing Financial Report.